modules/packages/msiexec.bat

```diff
@@ -0,0 +1,170 @@
+@echo off
+setlocal ENABLEDELAYEDEXPANSION
+
+rem Use real applications or testing stub for testing
+
+set MSIEXEC=msiexec
+set WMIC=wmic
+set CSCRIPT=cscript
+
+if not "%CFENGINE_MSIEXEC_TEST%"=="" (
+  set MSIEXEC=%CFENGINE_MSIEXEC_TEST% msiexec
+  set WMIC=%CFENGINE_MSIEXEC_TEST% wmic
+  set CSCRIPT=%CFENGINE_MSIEXEC_TEST% cscript
+)
+
+rem choose a function to execute
+
+if "%1"=="supports-api-version" <nul set /p=1
+if "%1"=="get-package-data"   goto :get_package_data_list
+if "%1"=="list-installed"     goto :list_installed
+if "%1"=="list-updates"       rem not implemented
+if "%1"=="list-updates-local" rem not implemented

```

Comment:
Should we return some sort of error or print a log message for "not implemented"?